phonon/update.py

```diff
@@ -88,16 +88,22 @@ def is_expired(self):
         return (current_time > self.hard_expiration or
                 current_time > self.soft_expiration)
 
+    def force_expiry(self):
+        """
```

Comment:
This one is a bit nit-picky, but maybe it would be better to rename this to `def set_force_expiry(self, value=True)` to allow for the value to be toggled.  Also,  `force_expiry` itself seems to indicate that this method will actually take some of action, which may cause confusion, since it is just setting the value.